README.md

```diff
@@ -1,32 +1,355 @@
+we learned how to clone arrays. Not surprisingly, the same advantages apply
+here. Plus, when you get to React in the next phase, you'll learn about the
+concept of _immutability_ and how central it is to developing using React. It's
+the same concept: when you write code that changes a variable, you should make a
+copy of the variable, and modify and return the copy rather than the original
+variable. Following this practice will make your React code more efficient and
+robust and make debugging easier.
+
+It is worth spending some time now making sure you're comfortable with cloning
+and modifying objects, because you'll use those skills frequently when you
+develop apps using React!
+
+## Modifying Objects
+
+Say we want to add a fourth princess, but we don't want to overwrite our
+original object. The first step is to clone the object, which we did above:
+
+```js
+const princessClone = { ...disneyPrincesses };
+```
+
+Let's log the clone to verify:
+
+```js
+console.log(princessClone);
+// => {The Little Mermaid: 'Ariel', Aladdin: 'Jasmine', Moana: 'Moana'}
+```
+
+Next, we'll add our fourth princess to `princessClone` using the assignment
+operator:
+
+```js
+princessClone.Tangled = "Rapunzel";
+console.log(princessClone);
+// => {The Little Mermaid: 'Ariel', Aladdin: 'Jasmine', Moana: 'Moana', Tangled: 'Rapunzel'}
+```
+
+We can also do both steps at the same time, by "spreading out" the properties in
+one object, adding a new property (or multiple properties) after that, and
+wrapping the whole thing in curly braces:
+
+```js
+const secondClone = { ...princessClone, "The Princess and the Frog": "Tania" };
+console.log(secondClone);
+// => {The Little Mermaid: 'Ariel', Aladdin: 'Jasmine', Moana: 'Moana', Tangled: 'Rapunzel', 'The Princess and the Frog": 'Tania'}
+```
+
+But we just noticed that we spelled our last princess's name wrong: it should be
+Tiana, not Tania. How can we fix that?
+
+### Modifying a Property of an Object
+
+One way we can fix our typo is by directly updating the property:
+
+```js
+secondClone["The Princess and the Frog"] = "Tiana";
+console.log(secondClone);
+// =>  {The Little Mermaid: 'Ariel', Aladdin: 'Jasmine', Moana: 'Moana', Tangled: 'Rapunzel', 'The Princess and the Frog": 'Tiana'}
+```
+
+Fixed! However, note that this change is _destructive_ — we've overwritten the
+value in the original object. As we've discussed, as a general rule, we want to
+make our updates nondestructively.
+
+For example, imagine we have an object that keeps track of accounts at a bank.
+We might have a `customer` variable that contains all the information about a
+particular customer:
+
+```js
+customer = {
+  name: "Jose Arroyo",
+  accountNumber: "0123456789",
+  accountType: "Checking",
+  balance: 1140.58,
+};
+```
+
+When the customer makes a deposit, we need a function to adjust the balance.
+Specifically, we need to:
+
+1. Create a clone of the variable so our function is pure,
```

Comment:
Stylistically, I would recommend keeping bullet points as separate sentences 

1. Create a clone of the variable so our function is pure.
2. Update the `balance` property of the clone to reflect the deposit. 
etc. etc.

I can't cite a source or guarantee truth because it came from a previous instructional designer I worked with, but I've learned that it's supposedly better for accessibility. I also just think it's better for general readability of bullet point lists.